velox/connectors/hive/HivePartitionName.h

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <memory>
+#include "velox/vector/ComplexVector.h"
+#include "velox/vector/SimpleVector.h"
+
+namespace facebook::velox::connector::hive {
+
+FOLLY_ALWAYS_INLINE std::string formatDate(int32_t value) {
+  return DATE()->toString(value);
```

Comment:
toString() APIs do not provide any guarantee about the format of the output, hence, should only be used for debugging / logging. Should this use something like toIso8601() ?